### DISCUSSION TITLE
Multiple mailserver for one service

### DISCUSSION THREAD

**Discussion by gaetan1903** [docker-mailserver/docker-mailserver#3382] [Q&A]
is it possible to configure two or more mail servers on a single service?
infact, I only have one server, and two of my domain names pointing to the same server, and I want to be able to configure it with a single service, without creating an additional port?

is it possible ?

thanks !

**Comment by polarathene** [comment]
I think you are asking if you can setup `@example.com` and `@something-else.com`?

That is fine, just add the accounts with `setup email add ...`. DMS is your mail server at `hostname: mail.whatever.com`, it does not have to be related to the email accounts you manage. Those domains use their DNS records to point to DMS as the mail server to use. No need to run multiple DMS for that.
